inksdk/ink/build.gradle

```diff
@@ -32,7 +32,7 @@ android {
 
     buildTypes {
         release {
-            minifyEnabled false
+            minifyEnabled true
```

Comment:
In order to even decrease more the size of the binary maybe you want to add/enable some rules in the proguard files.

Reply:
I will add it to the next version